autonomy/deploy/generators/docker_compose/templates.py

```diff
@@ -90,7 +90,6 @@
     container_name: abci{node_id}
     image: {runtime_image}
     environment:
-      - PYTHONHASHSEED=0
```

Comment:
I think we should keep this for all our envs

Reply:
Reverted change in 25abec60251a55b9d5e1a763c5b9be6290e8eef8.